web/src/main/java/com/epam/rft/atsy/web/controllers/rest/PositionController.java

```diff
@@ -48,15 +55,24 @@
 
     @ExceptionHandler(DuplicateRecordException.class)
     public ResponseEntity handleDuplicateException(Locale locale, DuplicateRecordException ex) {
+        HttpHeaders headers = new HttpHeaders();
+
+        headers.setContentType(TEXT_PLAIN_UTF8);
+
         return new ResponseEntity<>(messageSource.getMessage(DUPLICATE_POSITION_MESSAGE_KEY,
-                new Object[]{ex.getName()}, locale), HttpStatus.BAD_REQUEST);
+                new Object[]{ex.getName()}, locale), headers, HttpStatus.BAD_REQUEST);
     }
 
     @ExceptionHandler(Exception.class)
     public ResponseEntity handleException(Locale locale, Exception ex) {
         LOGGER.error("Error while saving position changes", ex);
+
+        HttpHeaders headers = new HttpHeaders();
```

Comment:
This section can be reused/extracted as well, but we will keep that work for the Spring MVC error handling task